indico/modules/rb_new/operations/blockings.py

```diff
@@ -57,16 +57,39 @@ def get_room_blockings(timeframe=None, created_by=None, in_rooms_owned_by=None):
     return query.all()
 
 
+def filter_blocked_rooms(blocked_rooms, overridable_only=False, nonoverridable_only=False,
+                         explicit=False):
+        if overridable_only:
+            blocked_rooms = [room for room in blocked_rooms
+                             if room.blocking.can_be_overridden(session.user, room.room, explicit_only=explicit)]
+        if nonoverridable_only:
+            blocked_rooms = [room for room in blocked_rooms
+                             if not room.blocking.can_be_overridden(session.user, room.room, explicit_only=explicit)]
+        return blocked_rooms
+
+
+def group_blocked_rooms(blocked_rooms):
+    return group_list(blocked_rooms, key=lambda obj: obj.room_id)
+
+
+def get_blockings(start_date, end_date):
+    return (BlockedRoom.query
+            .filter(BlockedRoom.state == BlockedRoomState.accepted,
+                    Blocking.start_date <= end_date,
+                    Blocking.end_date >= start_date)
+            .join(BlockedRoom.blocking)
+            .options(contains_eager('blocking'))).all()
```

Comment:
wrap `.all()` into the next line (ie keep it within the `()` surrounding everything)